src/org/nfrac/comportex/cells.cljc

```diff
@@ -434,8 +442,14 @@
                  (if good? (conj good-ids id) good-ids)
                  (if (and good? (< exc min-good-exc)) exc min-good-exc)))
```

Comment:
This isn't specific to this commit. But the winner dominates all other cells if it dominates... a single cell that's above the threshold? Shouldn't we instead just use filter/remove to get rid of the dominated cells case-by-case?

Alternately, if dominance is all-or-none, it should keep track of the second best excitation, not the lowest excitation above the threshold.

I might be looking at this wrong.